Algorithm.CSharp/RegressionTests/Universes/ETFConstituentUniverseFrameworkRegressionAlgorithm.cs

```diff
@@ -45,10 +46,8 @@ public override void Initialize()
             SetPortfolioConstruction(new ETFConstituentPortfolioModel());
             SetExecution(new ETFConstituentExecutionModel());
 
-            var spy = QuantConnect.Symbol.Create("SPY", SecurityType.Equity, Market.USA);
-
             UniverseSettings.Resolution = Resolution.Hour;
-            AddUniverse(Universe.ETF(spy, UniverseSettings, FilterETFConstituents));
+            AddUniverseSelection(new ETFConstituentsUniverseSelectionModel("SPY", universeFilterFunc: FilterETFConstituents));
```

Comment:
Suggest keeping the original version, if they give the same result sounds good to add a new one inheriting from this one just changing the initialize method 👍 and maybe removing the python for it `Languages`